lib/poker_hands/test_hands.txt

```diff
@@ -1 +1,3 @@
 royal_flush = [Card.new('14', 'D'), Card.new('13', 'D'), Card.new('12', 'D'), Card.new('11', 'D'), Card.new('10', 'D')]
+
+straight_flush = [Card.new('8', 'C'), Card.new('6', 'C'), Card.new('7', 'C'), Card.new('4', 'C'), Card.new('5', 'C')]
```

Comment:
Instead of defining these in a `.txt` file (and then presumably needing to parse this back into Ruby if we want to use it), we could consider creating these in a `.rb` file that contains our "test" objects.

Reply:
The point of doing this would be to allow us to reference them in code without copy pasting?

Reply:
Spoke a little IRL